packages/bpk-docs/src/layouts/NeoSideNavLayout/SectionsList.scss

```diff
@@ -40,7 +40,7 @@ $bpk-heading-button-padding: $bpk-spacing-md;
 
     @include bpk-text;
     @include bpk-text-xl;
-    @include bpk-border-bottom-sm($bpk-color-gray-700);
+    @include bpk-border-bottom-sm(var(--bpk-docs-sidebar-link-border, $bpk-color-gray-700));
```

Comment:
`var` doesn't work on IE, so we'll need to do both here:

```
@include bpk-border-bottom-sm($bpk-color-gray-700); // Fallback
@include bpk-border-bottom-sm(var(--bpk-docs-sidebar-link-border, $bpk-color-gray-700));
```

You don't need to do this when using `bpk-themeable-property` because that adds the fallback automatically.